README.md

```diff
@@ -124,3 +124,10 @@ You can then use `curl`to query the service:
 
     $ curl `minikube -n bb-readme service bb-readme-gateway-svc  --url`
     {"requestUid":"in:http-sid:point-to-point-channel-grpc:-1-h1:8080-66349706","payload":"BANANA"}
+
```

Comment:
Can you add a note and commands here to create the multiarch builder image? Something like what we have here: https://github.com/linkerd/linkerd2-proxy-init/blob/master/Makefile#L73